lib/library.rb

```diff
@@ -0,0 +1,35 @@
+#require 'Book'
+require 'Date'
+
+class Library
+ attr_accessor :books  # :duedate  #TODO:status
+
+  def initialize(library)
+    @books = []
+  #  @duedate = set_due_date
+  end
+
+ def check_in(book) #TODO library if its needed or not !!!
+    add = {item: book, status: :okay}
+    @books.push add
+  end
+
+  def lend(title,person)
+#    case
+#      when book[:status] = :checked_out then 'Sorry, book is already with another reader' #TODO can return the expected return date
```

Comment:
Metrics/LineLength: Line is too long. [132/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)